### ISSUE TITLE
duplicate key value violates unique constraint  asset_mint

### ISSUE THREAD

**Issue by ironmask285** [dcSpark/carp#195]
I encountered an error while running the index on preview.

```
2024-06-21T10:06:39.709754Z  INFO carp::sinks::cardano: Starting epoch 594 at block #2182304 (1e044c6be83cc31d3f8af2f1f0c8a232956a7592bd70cc43b99ed2070b76c92b)
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: Query("error returned from database: duplicate key value violates unique constraint \"asset_mint-pk\"")', indexer/tasks/src/multiera/multiera_asset_mint.rs:23:1
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: Query("error returned from database: current transaction is aborted, commands ignored until end of transaction block")', indexer/tasks/src/multiera/multiera_address.rs:28:1
```

please help

**Comment by SebastienGllmt** [comment]
What version of Carp are you using? I believe this issue should be fixed with `3.2.1`

**Comment by ironmask285** [comment]
yah, I upgraded to 3.2.1. I see it work fine. Thanks so muc @SebastienGllmt
